Dockerfile

```diff
@@ -1,48 +1,70 @@
-        g++ \
-        git \
-        iputils-ping \
-        make \
-        mysql-client \
-        libsqliteodbc \
-        locales \
-        odbc-postgresql \
-        postgresql-client \
-        sqlite3 \
-        unixodbc \
-        unixodbc-dev \
-        vim \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* \
- && ACCEPT_EULA=Y apt-get -qy install --no-upgrade --no-install-recommends \
-        msodbcsql \
-        mssql-tools \
- && echo "export PATH=$PATH:/opt/mssql-tools/bin" >> ~/.bash_profile \
- && echo "export PATH=$PATH:/opt/mssql-tools/bin" >> ~/.bashrc \
- && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
- && locale-gen \
- && odbcinst -i -d -f /usr/share/sqliteodbc/unixodbc.ini
+    && apt-get -qy install --no-upgrade --no-install-recommends \
+    apt-transport-https \
+    apt-utils \
+    clang-format-15 \
+    cmake \
+    curl \
+    g++ \
+    git \
+    gpg-agent \
+    iputils-ping \
+    libsqliteodbc \
+    libssl-dev \
+    locales \
+    make \
+    mysql-client \
+    odbc-postgresql \
+    postgresql-client \
+    software-properties-common \
+    sqlite3 \
+    unixodbc \
+    unixodbc-dev \
+    vim \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# NOTE: To install ODBC 18 on Ubuntu, see: https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
+#       Keep in mind that only certain releases of Ubuntu are supported.
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | tee /etc/apt/trusted.gpg.d/microsoft.asc \
+    && curl "https://packages.microsoft.com/config/ubuntu/$(lsb_release -rs)/prod.list" | tee /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get -qy update \
+    && ACCEPT_EULA=Y apt-get -qy install --no-upgrade --no-install-recommends \
+    msodbcsql18 \
+    mssql-tools18 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && echo "export PATH=$PATH:/opt/mssql-tools18/bin" >> ~/.bash_profile \
+    && echo "export PATH=$PATH:/opt/mssql-tools18/bin" >> ~/.bashrc \
+    && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
+    && locale-gen \
+    && odbcinst -i -d -f /usr/share/sqliteodbc/unixodbc.ini
+
+
+# NOTE: To install mysql-connector-odbc on Ubuntu, see: https://dev.mysql.com/downloads/connector/odbc/
+#       Keep in mind that only certain architectures are supported.
+RUN curl -SL "https://downloads.mysql.com/archives/get/p/10/file/mysql-connector-odbc-8.4.0-linux-glibc2.28-$(uname -m).tar.gz" -o mysql-connector-odbc.tar.gz \
+    && tar -xzf mysql-connector-odbc.tar.gz \
+    && rm mysql-connector-odbc.tar.gz \
+    && mv mysql-connector-odbc* /opt/mysql-connector-odbc \
+    && /opt/mysql-connector-odbc/bin/myodbc-installer -d -a -n "MySQL ODBC 8.4 ANSI Driver" -t "DRIVER=/opt/mysql-connector-odbc/lib/libmyodbc8a.so;" \
+    && /opt/mysql-connector-odbc/bin/myodbc-installer -d -a -n "MySQL ODBC 8.4 Unicode Driver" -t "DRIVER=/opt/mysql-connector-odbc/lib/libmyodbc8w.so;"
```

Comment:
The `DRIVER` paths were incorrect before this PR. I'm not sure this ever worked.